content/en/docs/allow-port-80.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Best Practice - Keep Port 80 Open
+permalink: /docs/allow-port-80
+top_graphic: 1
+date: 2019-01-24
+lastmod: 2019-01-24
+---
+
+{{< lastmod >}}
+
+We occasionally get reports from people who have trouble using the
+HTTP-01 challenge type because they've firewalled off port 80 to their
+web server. Our recommendation is that all servers meant for general web
+use should offer both HTTP on port 80 and HTTPS on port 443. They should
+also send redirects for all port 80 requests, and possibly an HSTS header.
```

Comment:
It might not be obvious to some readers that the HSTS header is only valid on port 443 (especially in this context, where you've just mentioned what they should do on port 80).